notebooks/06_Serving/06_02_kfserving_sdk.ipynb

```diff
@@ -0,0 +1,256 @@
+    "from kfserving import V1alpha2InferenceService\n",
+    "from kubernetes.client import V1ResourceRequirements"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Define namespace where InferenceService needs to be deployed to. If not specified, below function defines namespace to the current one where SDK is running in the cluster, otherwise it will deploy to default namespace."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "namespace = utils.get_default_target_namespace()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Define InferenceService"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Firstly define default endpoint spec, and then define the inferenceservice basic on the endpoint spec."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "api_version = constants.KFSERVING_GROUP + '/' + constants.KFSERVING_VERSION\n",
+    "default_endpoint_spec = V1alpha2EndpointSpec(\n",
+    "                          predictor=V1alpha2PredictorSpec(\n",
+    "                            tensorflow=V1alpha2TensorflowSpec(\n",
+    "                              storage_uri='gs://kfserving-samples/models/tensorflow/flowers',\n",
+    "                              resources=V1ResourceRequirements(\n",
+    "                                  requests={'cpu':'100m','memory':'1Gi'},\n",
+    "                                  limits={'cpu':'100m', 'memory':'1Gi'}))))\n",
+    "    \n",
+    "isvc = V1alpha2InferenceService(api_version=api_version,\n",
+    "                          kind=constants.KFSERVING_KIND,\n",
+    "                          metadata=client.V1ObjectMeta(\n",
+    "                              name='flower-sample', namespace=namespace),\n",
+    "                          spec=V1alpha2InferenceServiceSpec(default=default_endpoint_spec))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create InferenceService"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Call KFServingClient to create InferenceService."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "KFServing = KFServingClient()\n",
+    "KFServing.create(isvc)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Check the InferenceService"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "KFServing.get('flower-sample', namespace=namespace, watch=True, timeout_seconds=120)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Add Canary to InferenceService"
```

Comment:
Before this, we need to show how to invoke this endpoint.

Reply:
I can tell users how to invoke endpoint from users' local environment, but I can't find a way from SDK side.

Reply:
It's fine. Curl HTTP request will be helpful @PatrickXYS